### ISSUE TITLE
Cannot import supervised.automl

### ISSUE THREAD

**Issue by hokmingkwan** [mljar/mljar-supervised#476]
Hi there, after `!pip install mljar-supervised` , I tried to import `from supervised.automl import AutoML` , but I got this error `ModuleNotFoundError: No module named 'supervised'` 
Please let me know if you have any comments, thanks.

**Comment by sksuman97** [comment]
from supervised.automl import AutoML, not able to import. 
error-----> ModuleNotFoundError: No module named 'supervised'

**Comment by pplonski** [comment]
@sksuman97 have you installed `mljar-supervised`?

### Install command
```
pip install -U mljar-supervised
```

**Comment by cjgalvin** [comment]
After successfully installing mljar-supervised via conda, I had a similar issue. 
So I will share my problem and solution in case it helps.
**TLDR**: make sure your IDE is using the correct env.

Environment: M1 Mac, Python 3.9, VS Code, testing in Jupyter notebook

In my case, I had created a new Conda env, installed everything in there.
From terminal within VS Code, I activated this env.
However, when I launched a Jupyter notebook, the kernel was a previous env that did not have the mljar package.
Naturally, I could not import the module.
After a frustrating 15 minutes, I realized the error and activated the correct kernal via VS Code (cmd + P, select Interpreter; or in the upper right of the notebook pane in VS Code).
Worked afterward 🤦‍♂️

**Comment by Biku247** [comment]
I am having the same issue.
On Windows 11, I am creating a new anaconda environment with Python version 3.10.
Then using `conda install -c conda-forge mljar-supervised` this command to install.
It installs nicely but while importing through `from supervised.automl import AutoML`, it gives an error named `ModuleNotFoundError: No module named 'supervised'`
I was not able to find any solution online. Any possible solution will be highly appreciated.

**Comment by pplonski** [comment]
Hi @Biku247,

Please make sure that you install and import in the same virtual environment.

BTW, we are working on a solution that will help users install/import python packages, it can be done with one-click. Here is a screenshot of our new IDE. It will be released soon.
![image](https://github.com/mljar/mljar-supervised/assets/6959032/8145d77e-9208-40e5-a7c6-eace13181468)

**Comment by YuanfengZhang** [comment]
```bash
uv init mljar
cd mljar
uv venv --python=3.11
source .venv/bin/activate
uv pip install \
    -U \
    git+https://github.com/mljar/mljar-supervised.git@check-dependencies
python -c "from supervised.automl import AutoML"
```
returned:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'supervised'
```
